src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/ResetPasswordPolicyRequirement.cs

```diff
@@ -9,24 +10,41 @@ namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements
 public class ResetPasswordPolicyRequirement : IPolicyRequirement
 {
     /// <summary>
-    /// Indicates that new members will automatically be enrolled in account recovery administration when joining an organization.
+    /// List of Organization Ids that require automatic enrollment in password recovery.
     /// </summary>
-    public bool AutoEnrollEnabled { get; init; }
+    public IEnumerable<Guid> AutoEnroll { get; init; }
+
+    /// <summary>
+    /// Returns true if provided organizationId requires automatic enrollment in password recovery.
+    /// </summary>
+    public bool AutoEnrollEnabled(Guid organizationId)
+    {
+        return AutoEnroll.Contains(organizationId);
+    }
 }
 
 public class ResetPasswordPolicyRequirementFactory : BasePolicyRequirementFactory<ResetPasswordPolicyRequirement>
 {
     public override PolicyType PolicyType => PolicyType.ResetPassword;
 
+    protected override bool ExemptProviders => false;
+
+    protected override IEnumerable<OrganizationUserType> ExemptRoles => [];
+
     public override ResetPasswordPolicyRequirement Create(IEnumerable<PolicyDetails> policyDetails)
     {
         var result = policyDetails
-            .Select(p => p.GetDataModel<ResetPasswordDataModel>())
             .Aggregate(
-                new ResetPasswordPolicyRequirement(),
-                (result, data) => new ResetPasswordPolicyRequirement
+                new ResetPasswordPolicyRequirement() { AutoEnroll = [] },
+                (result, data) =>
                 {
-                    AutoEnrollEnabled = result.AutoEnrollEnabled || data.AutoEnrollEnabled
+                    var dataModel = data.GetDataModel<ResetPasswordDataModel>();
+                    if (dataModel.AutoEnrollEnabled && !result.AutoEnroll.Contains(data.OrganizationId))
+                    {
+                        return new ResetPasswordPolicyRequirement() { AutoEnroll = result.AutoEnroll.Append(data.OrganizationId) };
+                    }
+
+                    return result;
                 });
```

Comment:
This is basically filtering and then transforming a list. Using more granular linq operations that express each of these steps is easier to understand:

```c#
var orgsWithAutoEnroll = policyDetails
  .Where(p => p.GetDataModel().AutoEnrollEnabled)
  .Select(p => p.OrganizationId)
  .ToHashSet(); // to remove duplicates if you're concerned about them
```